packages/toolpad-app/src/runtime/constants.ts

```diff
@@ -1,3 +1,3 @@
 export const HTML_ID_EDITOR_OVERLAY = 'editor-overlay';
 
-export const NON_BINDABLE_CONTROL_TYPES = ['GridColumns'];
+export const NON_BINDABLE_CONTROL_TYPES = ['GridColumns', 'ChartData'];
```

Comment:
This feels like a leaky abstraction. Is there any reason we couldn't put a `bindable` property on the [`ArgControlSpec`](https://github.com/mui/mui-toolpad/blob/564108a247a30b6a570974308732020a6e1d479b/packages/toolpad-core/src/types.ts#L175C18-L175C32)?

_As a general remark, it's best to avoid `[].includes()` for lookup, better to use a `Set` instead for this purpose._

Reply:
Probably no reason, it does sound better so will try it.

Reply:
Also fine in a separate PR, it just caught my eye